src/components/templates/FilmographyDetail.js

```diff
@@ -24,7 +24,8 @@ class Filmography extends Component {
   }
 
   render() {
-    return (
+    try {
```

Comment:
what is your purpose to put try in render method? @alramdein

Reply:
I haven't added a guard (or I don't know what it's called) in routes for "if there is no `movie` data, users can't go to FilmographyDetail component." The problem is, FilmographyDetail l component can be accessed directly by typing the route on the URL bar. And of course, since there is no movie data passed from the FilmographyList component, it generates an error.  So I set `try` and `catch`.

Reply:
ah okey. but do that in route guard later. not within render function @alramdein